classroom.rb

```diff
@@ -0,0 +1,13 @@
+class Classroom
+  attr_accessor :label, :students
+
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  def add_student(student)
+    @students.push(student)
+    student.classroom = self
+  end
```

Comment:
- [ ] When setting the classroom for a student it also adds it to the classrooms' students, `a method that serves as the 'belongs-to' side (a student belongs to a classroom)` should be present in the `Student class`. Something like so: 

```ruby
  def classroom=(classroom)
      @classroom = classroom
      classroom.students.push(self) unless classroom.students.include?(self)
  end
```
Please kindly fix that.

Reply:
@t-yanick Thanks for your feedback!